assets/source/tests/tracking.test.js

```diff
@@ -0,0 +1,52 @@
+jest.mock( '@woocommerce/tracks', () => {
+	return {
+		recordEvent: jest.fn(),
+	};
+} );
+
+jest.mock( '@woocommerce/settings', () => {
+	return {
+		getSetting: () => {
+			return {
+				es: 'Spain',
+			};
+		},
+	};
+} );
+
+/**
+ * External dependencies
+ */
+import { recordEvent } from '@woocommerce/tracks';
+import { render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+/**
+ * Internal dependencies
+ */
+import UnsupportedCountryNotice from '../setup-guide/app/components/UnsupportedCountryNotice';
+import PrelaunchNotice from '../components/prelaunch-notice';
+
+describe( 'Component Records Events', () => {
+	it( 'UnsupportedCountryNotice records events on click', () => {
+		const { getByText } = render(
+			<UnsupportedCountryNotice countryCode="es" />
+		);
+
+		fireEvent.click( getByText( 'Change your store’s country here' ) );
+
+		expect( recordEvent.mock.calls[ 0 ][ 0 ] ).toBe(
+			'pfw_get_started_notice_link_click'
+		);
```

Comment:
:nail_care: 
WDYT of `toHaveBeenCalledWith`?
```suggestion
		expect( recordEvent ).toHaveBeenCalledWith(
			'pfw_get_started_notice_link_click',
			{
				context: 'pinterest-landing',
				href: '/wp-admin/admin.php?page=wc-settings',
				link_id: 'unsupported-country-link',
			}
		);
```
You can replace the values with `expect.any( String )` if you don't want to test the exact values.

Reply:
Fixed here https://github.com/woocommerce/pinterest-for-woocommerce/pull/299/commits/3ed9f9fda34e6fac23e807409e50a254a61766de

It should be `expect.any(Object)`

Reply:
> It should be `expect.any(Object)`

Or `{ context: expect.any( String ), href: expect.any( String ), link_id: expect.any( String ) }` ;)

But I'm fine with `.any( Object )` is we don't want to check the props.